src/types/venues.ts

```diff
@@ -33,6 +33,7 @@ export type GenericVenueTemplates = Exclude<
   | VenueTemplate.auditorium
   | VenueTemplate.viewingwindow
   | VenueTemplate.experiment
+  | VenueTemplate.artpiece
```

Comment:
[OOS] 

This looks like it would be shorter to replace the excluded ones with the included ones.

Considering the comment that follows further down
`which correctly narrows the types;`
a nice refactor could do interface segregation and then build up more complex types like lego bricks instead of exclude from an elaborated one.